src/v2i-hub/SpatPlugin/src/SpatPlugin.cpp

```diff
@@ -23,6 +23,10 @@ using namespace tmx::utils;
 namespace SpatPlugin {
 
 	SpatPlugin::SpatPlugin(const std::string &name) :PluginClientClockAware(name) {
+		if (isSimulationMode()) {
+			// Subscribe to the TimeSyncMessage
+			SubscribeToMessages();
```

Comment:
The same comment as above.

Reply:
No not on the SPAT plugin. See response above